README.md

```diff
@@ -32,12 +32,13 @@ $ go get -u go.einride.tech/aip-spanner
 Use a YAML config file to specify the schema to generate code from:
 
 ```yaml
-- name: music
-  schema:
-    - "testdata/migrations/music/*.up.sql"
-  package:
-    name: musicdb
-    path: ./internal/examples/musicdb
+databases:
+	- name: music
+		schema:
+			- "testdata/migrations/music/*.up.sql"
+		package:
+			name: musicdb
+			path: ./internal/examples/musicdb
```

Comment:
This looks a bit too indented, at least in Github diff.

Reply:
Must be some tab thing, I'll sed it